README.md

```diff
@@ -23,4 +27,6 @@ Note: You can add as many instances of this extension as you like!
 6. Update the existing or create a new manifest file (.trex) to point to the URL where you are hosting the extension with `/#/bookmarks` at the end. For example: `http://localhost:8080/#/bookmarks`.
 
 ## Support
-If you have questions about the extension or found a bug please open a new [issue](https://github.com/tableau/extension-filter-bookmarks/issues).
+Paid Tableau users can contact the Tableau Technical Support team for help.
```

Comment:
I'm not sure what a "Paid Tableau users" is? Do you mean Tableau users with a non free license? If there's a way to become a "Paid Tableau user" could you possibly link that as well?

Reply:
Maybe simpler to just say "Tableau customers can contact...", wdyt?

It just means people who are paying for Tableau can use our tech support.

Reply:
"Tableau customers can contact" sounds better. Do you happen to know if customers such as students that get a free license be included for tech support access? If not, it should be still fine.

Reply:
No it's just paid customers who can use technical support. I've updated the description with a new commit.

Reply:
Looks good